aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -21,6 +21,14 @@ const (
 	firehoseDeliveryStreamStatusDeleted = "DESTROYED"
 )
 
+const (
```

Comment:
I suggest prefixing these constants with `firehose` so that they don't at some point conflict with any constants in the package (plus it makes them private to the package).